pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -196,6 +201,34 @@ func hasUnresolvedImageStreamTag(g osgraph.Graph) bool {
 	return false
 }
 
+func hasCircularDependencies(g osgraph.Graph) bool {
+	// Filter out all but ImageStreamTag and BuildConfig nodes
+	var nodeFn osgraph.NodeFunc = func(g osgraph.Interface, n graph.Node) bool {
```

Comment:
This is going to be a common desire:  "build me a subgraph with nodes of these kinds and edges of these kinds".  Create a `NodeFunc` and an `EdgeFunc` that do that in `graph.go`, then I think I like it.